app/forms/organization_form.rb

```diff
@@ -0,0 +1,23 @@
+class OrganizationForm < YAAF::Form
+  attr_accessor :organization_attributes, :organization_membership, :current_user
+  after_save :create_organization_membership
+
+  def initialize(attributes)
+    super(attributes.except(:current_user))
```

Comment:
Maybe you can just:
```suggestion
    super(attributes)
```

and remove the line 8 below